File: internal/serial/usb_other.go
//go:build !darwin && !windows

package serial

// DetectMissingDrivers is a no-op on Linux for now. sysfs/udev enumeration
// would work — unimplemented until someone needs it.
func DetectMissingDrivers() ([]USBSerialCandidate, error) {
	return nil, nil
}

File: prototype/src/data/serial/usb_other.rs
//! Stub [`super::detect_missing_drivers`] for platforms without
//! bespoke USB enumeration logic (currently Linux + BSDs). sysfs +
//! udev enumeration would work here — unimplemented until someone
//! needs it.

use super::chipsets::USBSerialCandidate;

pub fn detect_missing_drivers() -> Result<Vec<USBSerialCandidate>, String> {
    Ok(Vec::new())
}
